src/app/footer/footer.component.scss

```diff
@@ -17,13 +17,13 @@
 }
 
 .footer {
-    
+
     color: white;
-    position: fixed;
+    // position: fixed;
```

Comment:
I think you do not want the footer fixed here.  Otherwise I think the footer would always be visible in the bottom portion of the screen,